tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml

```diff
@@ -408,13 +454,25 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
           <t>
                 The Service Proxy
                 function maps between App-flows and DetNet flows.
-				The DetNet Edge Node TSN function MUST support the TSN Stream
+				The DetNet Edge Node TSN entity MUST support the TSN Stream
                 identification functions and the related managed objects as
                 defined in <xref target="IEEE8021CB">IEEE 802.1CB</xref> and
                 <xref target="IEEEP8021CBdb">IEEE P802.1CBdb</xref> to
                 recognize the App-flow related packets.  The Service Proxy
                 presents TSN Streams as an App-flow to a DetNet Flow.
 		  </t>
+		<t>		
+			When a DetNet Service Proxy receives a packet from the TSN Entity
+			it MUST check whether such an App-flow is present in its mapping table.
+			If present it attaches the internal DetNet flow-ID to the packet and 

```

Comment:
May be "associates" would better than "attaches" as the internal DetNet Flow-ID is a node local identifier, used by the node internally. It is not visible outside.